### ISSUE TITLE
Other Sports

### ISSUE THREAD

**Issue by jbrumwell** [njacquelin/sports_field_registration#1]
Would this method work for hockey video?

**Comment by njacquelin** [comment]
Sure, I don't see why not. However you'll still have to create a hockey dataset for registration (ie map an image to a matrix).

**Comment by jbrumwell** [comment]
@njacquelin sorry I missed your comment, I would be interested to see if this would translate to hockey. I had a aicrowd challenge running but none of the resulting solutions worked. I am not sure if that dataset would work or if you would be interested in working on it? https://www.aicrowd.com/challenges/hockey-player-localization

Do you have an email, where we could communicate?

**Comment by njacquelin** [comment]
The method needs ground truth mapping between position on the image and position on the field. If I understand it correctly, the challenge you linked only contains information of players in the image, but not their position on the field. I don't think the method is suited for that semi-supervised task.

**Comment by jbrumwell** [comment]
@njacquelin correct, that is how I created the dataset for the challenge, I calculated the homography manually and translated the players positioning on the rink.  If I created a hockey dataset that mimics the swim dataset, do you see a reason why this solution wouldn't work?

**Comment by njacquelin** [comment]
I don't, it should work.
